corehq/apps/app_manager/models.py

```diff
@@ -2133,6 +2133,7 @@ class CaseSearch(DocumentSchema):
     default_properties = SchemaListProperty(DefaultCaseSearchProperty)
     blacklisted_owner_ids_expression = StringProperty()
     additional_case_types = ListProperty(str)
+    data_registry_id = StringProperty()
```

Comment:
rather than store the ID can we store the registry slug (which is unique and doesn't change). I think that makes it a bit more usable / debuggable.

Reply:
I suppose in this implementation you can put anything you like in here since it's just a plain input box

Reply:
I was initially planning on doing that, but slug is only unique on the domain, so we'd need both (assuming you can reference registries not hosted on your domain)

Reply:
Offlined - it's switched to globally unique [in the UI PR](https://github.com/dimagi/commcare-hq/pull/30150/files#diff-eb43997cd996632088d0b6a93a1f960929378523aa035dddc36acab197b31b31R70).  Will update to slug to reflect that.